es6-template-literals/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+    const street = `Irvine Center Drive`;
+    const address = `${number} ${street}`;
+
+    console.log('LearningFuze:', address);
```

Comment:
Please make sure you are A) doing a one-to-one conversion of the original code's concatenations, and B) not modifying any other parts of the code:

---

![image](https://user-images.githubusercontent.com/16143771/127085954-b4105072-6e96-42f0-94af-bcd63a73500b.png)

---

![image](https://user-images.githubusercontent.com/16143771/127085969-66d3b84d-fc29-48e2-93f6-add5e149937d.png)